pages/common/twine.md

```diff
@@ -0,0 +1,21 @@
+# twine
+
+> Twine is a utility for publishing Python packages on PyPI.
+> Subcommands such as `twine upload` have their own usage documentation.
```

Comment:
What is the difference between the current command and `twine-upload`? Is this line necessary?

Reply:
I used `pages/common/pip.md` as a template which had a line similar to this one.

`twine` has two other subcommands `check` and `register`, which I believe are seldom used. But perhaps I should include one example of each?

Reply:
> I used `pages/common/pip.md` as a template which had a line similar to this one.
> 
> `twine` has two other subcommands `check` and `register`, which I believe are seldom used. But perhaps I should include one example of each?

Thanks for the information, I will mark this conversation as resolved. As this line is applicable here.

Reply:
There does not seem to be a `tldr` page `twine upload` so it's weird to refer to it as if it existed.

I would get rid of this sentence here, and if applicable/useful, add one point for usage of `check` and `register` each.

If this was ever broken out, we could then add `Subcommands such as `twine upload` have their own usage documentation.` but right now this will just confuse users, as they won't find a `tldr twine upload` page.